src/sqlfluff/dialects/dialect_postgres_keywords.py~

```diff
@@ -0,0 +1,973 @@
+"""Keywords in the Postgres Dialect.
```

Comment:
This looks like a backup copy of dialect_postgres_keywords, please remove. General note: you need to be more conscious of what you're committing in these PRs

Reply:
Same idea. Sorry for that.